src/ui/public/directives/paginate.js

```diff
@@ -135,6 +136,17 @@ uiModules.get('kibana')
         } else {
           $scope.page = $scope.pages[0];
         }
+
+        //set initial value of paginationCount
+        self.updatePaginationCount(count ? 1 : 0);
+      };
+
+      self.updatePaginationCount = function (number) {
+        $rootScope.paginationCount = 0;
+        for(let i = 0; i < number; i++) {
+          $rootScope.paginationCount = $rootScope.paginationCount + $scope.pages[i].length;
+        }
+        return;
```

Comment:
Silly question: where is `paginationCount` used? What is this new logic doing? I think I must be missing something.